source/_integrations/script.markdown

```diff
@@ -24,7 +24,7 @@ script:
       # This is Home Assistant Script Syntax
       - service: notify.notify
         data_template:
-          message: Current temperature is {% raw %}{{ states('sensor.temperature') }}{% endraw %}
+          message: "Current temperature is {% raw %}{{ states('sensor.temperature') }}{% endraw %}"
```

Comment:
`{% raw %}` and `{% endraw %}` should be moved around the whole YAML block, and not be used inline.

I know you've just changed it because something else, so feel free to skip this.